### PR TITLE
adding Project function and update projects_controller

### PR DESCRIPTION
**Pull request by LeenMass**

### CODE REVIEW

**Review by MuhammadAw90 — DISMISSED** [review]

**Comment by LeenMass** [comment]
all the comments resolved
@tawfiknasser please re-review

**Comment by MuhammadAw90** [comment]
activating prettier wasn't that complicated.
Looks great!

**Review by Karyum — DISMISSED** [review]

**Comment by MuhammadAw90** [comment]
merging the previous branch caused conflict here, We're gonna fix them and request to merge again

**Comment by MuhammadAw90** [comment]
The conflict fixed

**Review by MuhammadAw90 — DISMISSED** [review]

File: controllers/projects.controller.js [hunk 1]
```diff
@@ -34,8 +35,25 @@ const showProjectsData = catchAsync(async (req, res) => {
     })
   }
 })
-
+const AddingProject = catchAsync(async (req, res) => {
```
Comment (tawfiknasser) [inline controllers/projects.controller.js]: aren't all controllers names camelCase ?

**Review by tawfiknasser — CHANGES_REQUESTED** [review]
use camelCase

**Review by MuhammadAw90 — APPROVED** [review]

**Review by Karyum — APPROVED** [review]

**Review by tawfiknasser — APPROVED** [review]

**Pull request merged**